### ISSUE TITLE
Wapm missing and qjs not run

### ISSUE THREAD

**Issue by reregaga** [wasmerio/wasmer#1971]
### Summary

I am trying to start learning and have run into difficulties.
Firstly, following the instructions from the page https://docs.wasmer.io/ecosystem/wasmer/getting-started, I cannot start the wapm, because it is suddenly not in release https://github.com/wasmerio/wasmer/releases/tag/1.0.0-rc1, although the instructions say that he is next to wasm. I had to compile wapm manually.
Secondly, when I launch qjs from the guide, I get an error and it is not clear what to do with it?
```
C:\Users\myprofile\wasmus>wasmer qjs.wasm
QuickJS - Type "\h" for help
qjs > thread 'main' panicked at 'not implemented: HostFile::poll in WasiFile is not implemented for non-Unix-like targets yet', lib\wasi\src\state\types.rs:403:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

### Additional details

Windows 10 Pro.

**Comment by Hywan** [comment]
Is the problem fixed @reregaga?

**Comment by MarkMcCaskey** [comment]
@Hywan we fixed the wapm release on Windows part, but the poll_oneoff being not fully implemented on Windows is still an issue. It won't take that long to fix, but I just don't have a lot of context on Windows syscalls... I spent a while looking for them before but I never got around to implementing it on Windows.

`poll_oneoff` is very posix-y but we should be able to get the "sleep" case working, which I think is what QJS really needs.

I'll file a separate issue for that and then put it in the backlog somewhere so we can get to it
